pkg/sql/sem/tree/type_check.go

```diff
@@ -204,19 +203,6 @@ func (sc *SemaContext) isUnresolvedPlaceholder(expr Expr) bool {
 	return sc.Placeholders.IsUnresolvedPlaceholder(expr)
 }
 
-// GetLocation returns the session timezone.
-func (sc *SemaContext) GetLocation() *time.Location {
-	if sc == nil || sc.Location == nil || *sc.Location == nil {
-		return time.UTC
-	}
-	return *sc.Location
```

Comment:
can we get rid of the struct methods here?

Reply:
sorry, not sure what you mean?

Reply:
does the `Location` variable on `semaCtx` need to exist still?

Reply:
Ah.. good point, removed.